src/test/java/com/basho/riak/client/core/operations/itest/ts/ITestFetchOperation.java

```diff
@@ -0,0 +1,57 @@
+package com.basho.riak.client.core.operations.itest.ts;
+
+import com.basho.riak.client.core.RiakFuture;
+import com.basho.riak.client.core.operations.ts.DeleteOperation;
+import com.basho.riak.client.core.operations.ts.FetchOperation;
+import com.basho.riak.client.core.operations.ts.StoreOperation;
+import com.basho.riak.client.core.query.timeseries.Cell;
+import com.basho.riak.client.core.query.timeseries.QueryResult;
+import com.basho.riak.client.core.query.timeseries.Row;
+import com.basho.riak.client.core.util.BinaryValue;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Time Series Fetch Operation Integration Tests
+ * @author Alex Moore <amoore at basho dot com>
+ * @since 2.0.3
+ */
+
+public class ITestFetchOperation extends ITestTsBase
+{
+    @BeforeClass
+    public static void InsertData() throws ExecutionException, InterruptedException
+    {
+        StoreOperation storeOp = new StoreOperation.Builder(tableNameBV).withRows(rows).build();
```

Comment:
It might be a good Idea to clean data before the test, but considering the fact that we can't completely restore initial RiakTS state (we can't drop the bucket type, so it can has the unexpected schema) we may live w/o this